### PR TITLE
Stanage Pilot FAQ shift and tidy up.

### PR DESCRIPTION
**Pull request by jkwmoore**
Deletes the Stanage pilot section and brings the important FAQ items into the main Stanage tree.

Supersedes: https://github.com/rcgsheffield/sheffield_hpc/pull/1757/files (includes the changes from this PR.)

### CODE REVIEW

**Comment by jkwmoore** [comment]
n.b Additional labels for each section within the FAQ page may be useful but aren't needed yet.

File: stanage/stanage-FAQ-gotchas.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.. _stanage-FAQ-gotchas:
+
+Stanage FAQ and Gotchas
```
Comment (willfurnass) [inline stanage/stanage-FAQ-gotchas.rst:3]: Still ideally want a better name/title for this page

File: stanage/index.rst [hunk 2]
```diff
@@ -20,17 +20,21 @@ Stanage
 This is the documentation for Stanage, the University of Sheffield's new High Performance Computing cluster. Stanage is the logical successor to the :ref:`sharc` cluster featuring 
 more modern CPUs with significantly greater numbers of cores available.
 
+It is highly recommended that users read the :ref:`stanage-FAQ-gotchas` and :ref:`stanage-totp-setup` pages as these contain vital information for users new to the cluster.
```
Comment (willfurnass) [inline stanage/index.rst:23]: make a `.. note::`?

**Review by willfurnass — APPROVED** [review]
LGTM bar a better name/title for that page.

**Comment by jkwmoore** [comment]
Now looks like the below as the 1st line / end users appear to be looking to the main Stanage page to resolve issues when unable to connect on their first connection:

image

**Review by willfurnass — APPROVED** [review]

File: stanage/stanage-FAQ-gotchas.rst (hunk 1 above)
Comment (jkwmoore) [reply stanage/stanage-FAQ-gotchas.rst:3]: I'll merge this for now and we can make subsequent amendment to the title in a bit.

**Pull request merged**